doc/webui/README.md

```diff
@@ -0,0 +1,66 @@
+# elektra web
+
+_a web user interface (Web UI) to remotely manage multiple elektra instances_
+
+
+## Overview
+
+Elektra web consists of multiple components:
+
+ * (multiple) servers running an elektra daemon (`elektrad`)
+ * a single cluster management server to communicate with the elektra daemons (`clusterd`)
```

Comment:
I think its better to remove the architecture parts from here and put them into an extra document as discussed. This document should only be about the HTTP API.